reset-db.sh

```diff
@@ -0,0 +1,501 @@
+        fi
+      done
+#      if test -f "$DMSS_SYSTEM"; then
+#          echo "  Updating dmss-system.radix.json"
+#          sed -i "$SED_PATTERN" "$DMSS_SYSTEM"
+#          grep -Eq "$GREP_PATTERN" "$DMSS_SYSTEM" && ok || err
+#      else
+#        warn "    The file does not exist"
+#      fi
+    else
+      fatal "- Missing required variable 'MONGO_AZURE_USER'. Exiting."
+    fi
+}
+
+function set_database_password() {
+    SED_PATTERN="s/\"password\":.*\",/\"password\": \"$MONGO_AZURE_PW\",/"
+    GREP_PATTERN="^\s{1,}\"password\": \"$MONGO_AZURE_PW\","
+
+    info "Setting database passwords.."
+    if [ -n "$MONGO_AZURE_PW" ]; then
+      for data_source in "${DATA_SOURCES[@]}"; do
+        echo "  Updating $data_source"
+        if test -f "$data_source"; then
+          sed -i "$SED_PATTERN" "$data_source"
+          grep -Eq "$GREP_PATTERN" "$data_source" && ok || err
+        else
+          warn "    The file does not exist"
+        fi
+      done
+#      if test -f "$DMSS_SYSTEM"; then
+#          echo "  Updating dmss-system.radix.json"
+#          sed -i "$SED_PATTERN" "$DMSS_SYSTEM"
+#          grep -Eq "$GREP_PATTERN" "$DMSS_SYSTEM" && ok || err
+#      else
+#        warn "    The file does not exist"
+#      fi
+    else
+      fatal "- Missing required variable 'MONGO_AZURE_PW'. Exiting."
+    fi
+}
+
+function set_data_source_names() {
+  info "Removing 'Test'-prefix from data source names.."
+  for data_source in "${DATA_SOURCES[@]}"; do
+    echo "  Updating $data_source"
+    if test -f "$data_source"; then
+      if grep -Eq '"name": "Test.*",' "$data_source"; then
+        sed -i -E "s/\"name\": \"Test/\"name\": \"/" "$data_source"
+      fi
+      grep -Eq '"name": "Test.*",' "$data_source" && err || ok
+    fi
+  done
+}
+
+#function update_compose_spec() {
+#  TARGET_ENV="ENVIRONMENT: production"
+#  TARGET_LOG="LOGGING_LEVEL: debug"
+#  info "Updating compose spec.."
+#  if test -f "$COMPOSE_FILE"; then
+#    echo "  Updating volume mount"
+#    sed -i "s/dmss-system.local.json:/dmss-system.radix.json:/" "$COMPOSE_FILE"
+#    grep -Eq "^\s{6}- ./dmss-system.radix.json:" "$COMPOSE_FILE" && ok || err
+#
+#    echo "  Updating ENVIRONMENT.."
+#    sed -i "s/ENVIRONMENT: \${DMSS_ENVIRONMENT:-local}/${TARGET_ENV}/" "$COMPOSE_FILE"
+#    dmss_service_spec=$(grep -EA 20 "^\s{2}dmss:" "$COMPOSE_FILE")
+#    echo "$dmss_service_spec" | grep -Eq "^\s{6}${TARGET_ENV}" && ok || err
+#
+#    echo "  Updating LOGGING_LEVEL.."
+#    sed -i "s/LOGGING_LEVEL: \".*\"$/${TARGET_LOG}/g" "$COMPOSE_FILE"
+#    dmss_service_spec=$(grep -EA 20 "^\s{2}dmss:" "$COMPOSE_FILE")
+#    echo "$dmss_service_spec" | grep -Eq "^\s{6}${TARGET_LOG}" && ok || err
+#  fi
+#}
+
+function build_images() {
+  info "Building the Docker images.."
+  docker-compose build --quiet api  && ok || err
+}
+
+function dmss_reset_app() {
+  info "Resetting DMSS.."
+  if [ "$DRY_RUN" == "False" ]; then
+    cd ../data-modelling-storage-service
```

Comment:
this part is changed compared to FoR: must cd to dmss folder and run the reset-app command